query-accelerator/latest/config/index.md

```diff
@@ -0,0 +1,241 @@
+    },
+    {
+      "name": "cm:author",
+      "isIndex": true
+    }
+  ],
+  "aspects": [
+    {
+      "name": "cm:titled",
+      "isIndex": true
+    }
+  ],
+  "compositeIndexes": {
+    "index_1": ["cm:name", "cm:author"],
+    "index_2": ["cm:name", "cm:titled"]
+  }
+}
+```
+
+ACS node properties:
+![acs-node-properties]({% link query-accelerator/images/acs-properties.png %})
+Table entry:
+
+| node_id | owner_id | alf_type | cm_name   | cm_author   | cm_titled |
+| ------- | -------- | -------- | --------- | ----------- | --------- |
+| 887     | 3        | 24       | demo1.txt | Joe Bloggs  | true      |
+
+### Example 2
+The following example requires that at least one node in the system has the DublinCore aspect applied. This aspect is
+part of the system by default. If there are no nodes, an unknown property error is reported.
+
+```json
+{
+  "version": 6,
+  "name": "ac",
+  "properties": [
+    {
+      "name": "cm:name",
+      "isIndex": true
+    },
+    {
+      "name": "cm:publisher",
+      "isIndex": true
+    }
+  ],
+  "aspects": [
+    {
+      "name": "cm:titled",
+      "isIndex": true
+    },
+    {
+      "name": "cm:dublincore",
+      "isIndex": true
+    }
+  ],
+  "compositeIndexes": {
+    "index_1": ["cm:name", "cm:publisher"],
+    "index_2": ["cm:name", "cm:titled"]
+  }
+}
+```
+
+ACS node properties and aspects:
+![acs-node-aspects]({% link query-accelerator/images/acs-aspects.png %})
+Table entry:
+
+| node_id | owner_id | alf_type | cm_name   | cm_publisher | cm_titled | cm_dublincore |
+| ------- | -------- | -------- | --------- | ------------ | --------- | ------------- |
+| 918     | 3        | 24       | demo2     | Egmont       | true      | true          |
+
+## Query Sets and Transaction Meta-Data Queries (TMDQ)
+
+Here we give an example of how to create a Query Set to replace a TMDQ.
+
+The following TMDQ selects all documents (cm:content) which have a dublincore aspect (cm:dublincore), a
+publisher (cm:publisher) equal to 'Hachette Livre' and a type (cm:type) equal to 'Action'.
+
+```
+{
+   "query":{
+      "query":"select * from cmis:document as d join cm:dublincore as dc on d.cmis:objectId = dc.cmis:objectId where dc.cm:publisher = 'Hachette Livre' and dc.cm:type='Action'",
+      "language":"cmis"
+   }
+}
+```
+The following Query Set would be able to support the above TMDQ. It also requires that at least one node in the system
+has the DublinCore aspect applied.
+
+```json
+{
+  "version": "1",
+  "name": "doc_dublincore",
+  "properties": [
+    {
+      "name": "cm:publisher",
+      "isIndex": true
+    },
+    {
+      "name": "cm:type",
+      "isIndex": true
+    }
+  ],
+  "aspects": [
+    {
+      "name": "cm:dublincore",
+      "isIndex": true
+    }
+  ]
+}
+```
+
+## Query Set Status and Caching
+
+Denormalized tables have a status. For example:
+
+| Name | Version | State | Notes |
+| ---- | ------- | ----- | ----- |
+| tableA | 1 | OBSOLETE | About to be removed |
```

Comment:
"About to be removed" should be replaced with "Can be removed" or similar.
Also need an extra row in here:
| tableA | 2 | RETIRED | Previous LIVE version. Can be removed |

Reply:
done